networks/http/http.go

```diff
@@ -72,7 +72,7 @@ func NewNetwork(host string, chainHash string) (*Network, error) {
 		return nil, ErrNotUnchained
 	}
 
-	if !(sch.Name == crypto.UnchainedSchemeID || sch.Name == crypto.ShortSigSchemeID) {
+	if !(sch.Name == crypto.UnchainedSchemeID || sch.Name == crypto.ShortSigSchemeID || sch.Name == crypto.SigsOnG1ID) {
```

Comment:
This feels like a meta issue maybe for our drand repo. Or for governance of the League of Entropy.

This are just exported variable name, I'm happy to change them in the drand/drand repo if we settle on a convention, but they are just a shortcut to access the actual network names which are fixed for the existing networks, I'm afraid since changing them would change the chain hashes.